packages/viewer/src/components/viewer-markup-arrow/viewer-markup-arrow.css

```diff
@@ -0,0 +1,138 @@
+:host {
+  font-size: 0.75rem;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  left: 0;
+  top: 0;
+  pointer-events: none;
```

Comment:
Please review these properties. On first look, `font-size`, `overflow`, `left`, `top` and `pointer-events` are not needed.

Reply:
Looks like `font-size` and `overflow` are not needed, but `left` and `top` are needed to prevent the arrow from appearing below the viewer, and `pointer-events` is necessary to allow the `pointerdown` from the viewer to trigger the listener in the arrow/circle markup